sdks/java/core/src/test/java/com/google/cloud/dataflow/sdk/transforms/display/DisplayDataMatchers.java

```diff
@@ -16,13 +16,19 @@
 
 package com.google.cloud.dataflow.sdk.transforms.display;
 
+import static org.hamcrest.Matchers.allOf;
```

Comment:
Today we have no user-facing APIs that link with Hamcrest AFAIK. I think we are trying to sort out a good way to do that without causing dependency hell. (We redacted `SerializableMatchers` to be package-private because of this issue; we have a lot of value to add by solving it).

It looks like you might be able to get away with submitting it package-private for now.